sdk/communication/azure-communication-sms/pom.xml

```diff
@@ -52,10 +52,6 @@
         <artifactId>jacoco-maven-plugin</artifactId>
         <version>0.8.5</version> <!-- {x-version-update;org.jacoco:jacoco-maven-plugin;external_dependency} -->
         <configuration>
-          <properties>
```

Comment:
This whole jacoco plugin can be removed.

Reply:
Addressed, thank you!